nestkernel/event_delivery_manager.cpp

```diff
@@ -736,7 +729,6 @@ EventDeliveryManager::gather_target_data( const thread tid )
         buffer_size_target_data_has_changed_ = kernel().mpi_manager.increase_buffer_size_target_data();
       }
     }
-#pragma omp barrier
```

Comment:
Agree here: Superfluous barrier because of preceding invocation of gather_completed_checker_.any_false().